modules/virt-configuring-live-migration-limits.adoc

```diff
@@ -25,18 +25,32 @@ $ oc edit configmap kubevirt-config -n openshift-cnv
 [source,yaml]
```

Comment:
Possible spot for an `.Example output` [block title](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output)?

Reply:
Added block title  `.Example configuration file`